bgpd/bgp_route.c

```diff
@@ -5364,7 +5364,7 @@ void bgp_aggregate_increment(struct bgp *bgp, struct prefix *p,
 
 	/* MPLS-VPN aggregation is not yet supported. */
 	if ((safi == SAFI_MPLS_VPN) || (safi == SAFI_ENCAP)
-	    || (safi == SAFI_EVPN))
+	    || (safi == SAFI_EVPN) || (safi == SAFI_FLOWSPEC))
```

Comment:
I prefer every SAFI to be on a new line, so code review and merging is easier whenever we add a SAFI. Historically,  it was not done, but let's have a good practice from now, please.

Reply:
ok

Reply:
I believe @vjardin6WIND has asked this to be put on it's own line, yet it appears it is still here